stylesheets/_index.scss

```diff
@@ -24,7 +24,7 @@
   .content {
     background-color: $grey_l;
     flex-grow: 1;
-    overflow-y: auto;
+    overflow-y: overlay;
```

Comment:
Here's the bug tracking removal from WebKit, which is marked WontFix: https://bugs.chromium.org/p/chromium/issues/detail?id=554361 It also links to the csswg issue tracking a potential standard for this behavior.